spec/fixtures/cookbooks/oneview_test_api500_synergy/metadata.rb

```diff
@@ -0,0 +1,7 @@
+name             'oneview_test_api500_synergy'
+maintainer       'none'
+license          'All rights reserved'
+description      'A test cookbook for the oneview cookbook API500::Synergy module'
+version          '0.1.0'
```

Comment:
Shouldn't it be '1.0.0' ? Any point starting it on 0.1.0?

Reply:
This is the fake cookbook for testing